doc/source/dev/missing-bits.rst

```diff
@@ -0,0 +1,260 @@
+Function or class              Variable name
+============================== =============
+`scipy.stats.boxcox`           ``lmbda``
+`scipy.stats.boxcox_llf`       ``lmb``
+`scipy.stats.yeojohnson`       ``lmbda``
+`scipy.stats.yeojohnson_llf`   ``lmb``
+`scipy.stats.chi2_contingency` ``lambda_``
+`scipy.stats.boltzmann`        ``lambda_``
+`scipy.stats.planck`           ``lambda_``
+`scipy.stats.tukeylambda`      ``lam``
+`scipy.signal.cspline1d`       ``lamb``
+`scipy.signal.qspline1d`       ``lamb``
+`scipy.signal.cspline2d`       ``lambda`` [*]
+`scipy.signal.qspline2d`       ``lambda`` [*]
+`scipy.signal.spline_filter`   ``lmbda``
+`scipy.special.tklmbda`        ``lmbda``
+`scipy.special.boxcox`         ``lmbda``
+`scipy.special.boxcox1p`       ``lmbda``
+`scipy.special.inv_boxcox`     ``lmbda``
+`scipy.special.inv_boxcox1p`   ``lmbda``
+============================== =============
+
+.. [*] This function accepts positional arguments only, but the docstring
+       documents the argument as `lambda`.
+
+
+Implicit string continuation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+When an implicit continuation is used to create a long literal string, the
+string is generally split at a space, and the space is included in the
+second part of the string.
+
+  Yes::
+
+      long_string = ("This is a very long string that demonstrates"
+                     " this guideline.")
+
+  No::
+
+      long_string = ("This is a very long string that demonstrates "
+                     "this guideline.")
+
+
+Documentation Guidelines
+------------------------
+
+Use of LaTeX
+~~~~~~~~~~~~
+[TBD.  In the past, we had an unwritten rule that LaTex markup should be
+restricted to the Notes section.  We don't follow that rule these days.
+Should we?]
+
+
+Use "must", not "should"
+~~~~~~~~~~~~~~~~~~~~~~~~
+When specifying a required condition on the input parameters, the
+word "must" is preferable to "should".  For many English speakers,
+"must" implies a stronger constraint than "should",  e.g. "I must
+have oxygen to live" versus "I should exercise more".
+
+    Yes::
+
+            Parameters
+            ----------
+            x : float
+                x must be nonnegative.
+
+    No::
+
+            Parameters
+            ----------
+            x : float
+                x should be nonnegative.
+
+
+Use of the 'versionadded' markup
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* For a new function, the 'versionadded' markup goes in the "Notes" section,
+  *not* in the description at the beginning of the docstring.
+* For a new argument added to an existing function,  two locations have been
+  used for the the 'versionadded' markup, [TBD: which is preferred?]:
+
+  * At the end of the description of the argument in the "Parameters" section
+  * In the "Notes" section.  In this case, the `versionadded` markup
+    wouldn't be used.  Instead, the new addition is noted with a plain
+    text comment.
+
+
+Citing wikipedia articles in the "References" section
```

Comment:
I would also add the preferred way to cite articles. Lately I have used: `Fang et al. "Design and modeling for computer experiments". Computer Science and Data Analysis Series, 2006.`

Reply:
There are so many different conventions for this, and I have no preference, so for now I'm not going to try to provide general guidelines for the reference format.  Perhaps that can wait for a follow-up PR.

Reply:
Use of DOI must be emphasized over never-ending natbib or ieeetran discussions. DOIs are here to stay the rest is constantly changing.

Reply:
Agreed and I discovered recently we have special syntax for this and arXiv.

Reply:
I'll add a note about recommending the use of DOIs, and mention the available markup.